pkgs/flox-pkgdb/default.nix

```
@@ -183,12 +183,12 @@ in
           lcov
           remake
           # For IDEs
-          ccls
-          bear
+          # ccls
+          # bear
           # For lints/fmt
-          clang-tools_16
-          include-what-you-use
-          llvm # for `llvm-symbolizer'
+          # clang-tools_16
```

we still include clang-tools for the formatter git hook

can we extract `clang-{format,tidy}` from this?

```suggestion
          clang-tools_16
```